dbt/adapters/bigquery/connections.py

```diff
@@ -649,6 +646,16 @@ def _query_and_results(
         # Cannot reuse job_config if destination is set and ddl is used
         job_config = google.cloud.bigquery.QueryJobConfig(**job_params)
         query_job = client.query(query=sql, job_config=job_config, timeout=job_creation_timeout)
+
+        if (
+            query_job.location is not None
+            and query_job.job_id is not None
+            and query_job.project is not None
+        ):
+            logger.debug(
```

Comment:
hey Christophe! what do you think about making this an `info` level log?

this job URL is super useful even beyond debugging code, since it's the ground truth of what logic and process is actually executing the model. I think it should be more readily available, understanding that most apps out there probably wouldn't enable `debug` logs in production

Reply:
Hello @robsicurelli, I agree with you that it's making the logs unnecessarily verbose if you'd like to keep BQ links in your scheduler logs all the time.
I'm fine moving it to `info` but I'll let @Fleid call the shot on that one 👍

Reply:
@Fleid that sounds good to me. I'm happy to stamp this PR as-is, and open another issue for changing the log level